source/declarators.tex

```diff
@@ -2139,7 +2139,7 @@
 \grammarterm{function-body}
 is of the form
 \tcode{= default ;}
-is called an \grammarterm{explicitly-defaulted} definition.
+is called an \techterm{explicitly-defaulted} definition.
```

Comment:
Change this to `\defnx{explicitly-defaulted}{definition!function!explicitly-defaulted}`, and remove the `\indextext` above.